sympy/utilities/iterables.py

```diff
@@ -500,7 +500,7 @@ def subsets(seq, k=None, repetition=False):
            [(1, 2), (1, 3), (2, 3)]
 
 
-       subsets(seq, k, repetition=True) will return the (n - 1 + k)!/k!/(n - 1)!
+       ``subsets(seq, k, repetition=True)`` will return the `\\frac{(n - 1 + k)!}{k!(n - 1)!}`
```

Comment:
Should this (and the similar instance above) be \`\frac ...\`, now that the docstring is prefixed by `r"""`?